charts/certmanager/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: letsencrypt
```

Comment:
This does not match the name of the Chart

Reply:
Thanks for noticing it, updated the chart to reflect valid name.